app/routes/labs/parallax/ParallaxImage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react'
+
+// Define an interface for the component's props
+interface ParallaxImageProps {
+  src: string // URL of the image
+  alt: string // Description of the image
+}
+
+function ParallaxImage({ src, alt }: ParallaxImageProps) {
+  const [offset, setOffset] = useState(0)
+
+  const handleScroll = () => {
+    const newOffset = window.pageYOffset
+    setOffset(newOffset)
+  }
+
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll)
+    return () => window.removeEventListener('scroll', handleScroll)
+  }, [])
+
```

Comment:
Did you copy this from somewhere? If so, it would be preferable to add it to package.json as a dependency. If that's not possible, then we need to reproduce any relevant license information.

Also, it would be preferable to organize this as a reusable component, rather than hardcoding its content.

Reply:
Nope, this was my own creation after looking at how various other image related components were structured.  

I've restructured the code to allow the title, paragraph text, and image to be specified as properties to make it a reusable component.